shopify/resources/application_credit.py

```diff
@@ -0,0 +1,12 @@
+from ..base import ShopifyResource
+
+class ApplicationCredit(ShopifyResource):
+    _prefix_source = "application_credits/$application_credit_id/"
```

Comment:
I don't believe you need to specify this and add logic for the _prefix, pyactiveresource will default to `application_credits/$id` naturally. A `pass` here should still work

Reply:
+1 for `pass`, check out the implementation in the Ruby client library: https://github.com/Shopify/shopify_api/blob/master/lib/shopify_api/resources/application_credit.rb

`pyactiveresource` should be able to do all of the heavy lifting here with no special logic

Reply:
Okay thank you so much @mllemango @jamiemtdwyer